pytorch_lightning/accelerators/accelerator.py

```diff
@@ -348,7 +348,12 @@ def connect_precision_plugin(self, plugin: PrecisionPlugin) -> None:
 
     def to_device(self, batch: Any) -> Any:
         """Pushes the batch to the root device"""
-        return self.batch_to_device(batch, self.root_device)
+        # Todo (tchaton) Better fix
+        is_dict = isinstance(batch, dict)
+        if is_dict:
+            batch = [batch]
```

Comment:
```suggestion
		batch = [batch] if isinstance(batch, dict) else batch
```

Reply:
I need to unwrap the dictionary from the list. The boolean is used to keep track of this. But a better fix should be found for using transformers library with Lightning.